src/etl/LoadBalancer.cpp

```diff
@@ -151,6 +151,13 @@ LoadBalancer::LoadBalancer(
             ));
         } else {
             etlState_ = stateOpt;
+            /*
+             * NOTE: Rippled NetworkID: Mainnet = 0; Testnet = 1; Devnet = 2
+             * However, if rippled is running on neither of these (ie. standalone mode) rippled will show 0, but
+             * is not included in the stateOpt response. Must manually add it here.
+             */
+            if (!etlState_->networkID.has_value())
+                etlState_->networkID = 0;
```

Comment:
You could also default `networkID` to 0 in `ETLState` even though it's an optional. That would move this comment to ETLState (not sure if that's better) and allow you to rollback the multiple one line changes that create `ETLState{.networkID = 0}`.
There also might be some constant in libXRPL that we can use to get this `0` for mainnet.